src/PayloadParser.h

```diff
@@ -394,7 +394,7 @@ namespace snowcrash {
                                    const ParseResultRef<Payload>& out) {
 
             const char* regex;
-            mdp::ByteBuffer mediaType;
+	    mdp::ByteBuffer mediaType;
```

Comment:
You are right, I should restart emacs once in a while, the automatic formatting on save got messed up somehow.